src/components/CreateVmWizard/helpers.js

```diff
@@ -163,12 +165,32 @@ const verifyOsIdToCluster = (selectedOsId, clusterId, { clusters, operatingSyste
   return selectedOs ? selectedOs.get('id') : '0'
 }
 
+const isTpmRequired = (operatingSystemId, operatingSystems) => {
+  return operatingSystems.getIn([operatingSystemId, 'tpmSupport']) === 'required'
+}
+
+const getTpmChange = (operatingSystemId, operatingSystems) => {
```

Comment:
Why declaring this method here and not in src/helpers.js.
It is not really used by the createVmWizard.

Reply:
no particular reason - we can change it